internal/image/image.go

```diff
@@ -268,6 +268,7 @@ func inWhiteoutDir(fileMap Layer, filePath string) bool {
 		if filePath == dirname {
 			break
 		}
+
 		node := fileMap.fileNodeTrie.Get(dirname)
```

Comment:
nit: I actually agree with this change, but since it's the only one in this file, removing it would take a while file out of the diff 😅 

```suggestion
		}
		node := fileMap.fileNodeTrie.Get(dirname)
```